homeassistant/components/fordpass/config_flow.py

```diff
@@ -0,0 +1,76 @@
+"""Config flow for FordPass integration."""
+import logging
+
+from fordpass import Vehicle
+import voluptuous as vol
+
+from homeassistant import config_entries, core, exceptions
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME
+
+from .const import DOMAIN, VIN  # pylint:disable=unused-import
+
+_LOGGER = logging.getLogger(__name__)
+
+DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_USERNAME): str,
+        vol.Required(CONF_PASSWORD): str,
+        vol.Required(VIN): vol.All(str, vol.Length(min=17, max=17)),
+    }
+)
+
+
+async def validate_input(hass: core.HomeAssistant, data):
```

Comment:
Is it possible to set a config flow unique id so that an account can only be linked once?
It can be an account number or a username if it is unique.
Note that the unique id is case-sensitive.
https://developers.home-assistant.io/docs/config_entries_config_flow_handler#unique-ids

If you don't add a unique id the `already_configured` string is never used so delete it.